variable/energy/energy-final.yaml

```diff
@@ -273,3 +273,38 @@ Final Energy|Transportation|Liquids|Oil:
    description: Final energy consumption by the transportation sector of liquid
                 oil products (from conventional & unconventional oil)
    unit: EJ/yr
+
+Final Energy|Electricity|Profile:
```

Comment:
I'm wondering why this needs an own variable? If the variable `Final Energy` is given for a full year, it is the total consumption over the year - if it is given with an hourly disaggregation, it already fulfils the role of this "profile". So I see this as a duplication.

Reply:
@danielhuppmann I made a mistake here with the unit. This is not MWh but percentages and this variable is kind of a loadfactor (if you multiply hourly values by yearly MWh you get hourly MWh). The idea is that those profiles are "fixed" and they include all the seasonal, weekly, daily variations as well as the stochasticity (they are scenarised). When performing a case study, the real thing that is put into the plan4EU core is profile multiplied by yearly energy. Goig to correct that.

Reply:
Thanks for the clarification @sandrinecharousset! If this is a relative value, please clarify as a `note` attribute what the 100%-value is (average load, peak load).

Reply:
@danielhuppmann I have just commited a change here

Reply:
@danielhuppmann I changed it; added a note attribute as suggested, and regarding the unit, I think that is is not either a percentage as 100% does not have a meaning. Those time series are time series of coefficients which, when multiplied by a yearly energy in MWh, give time series of MWh. I wrote that in the note.